tests/test_regression_against_cases.py

```diff
@@ -24,6 +24,10 @@ def test_regression_against_case(case: Path):
     dataset = read_dataset_from_netcdf(Path(__file__).parent / "regression_results" / f"test1_{case.parent.stem}.nc").load()
     reference_dataset = read_dataset_from_netcdf(Path(__file__).parent / "regression_results" / f"{case_name}_result.nc").load()
 
+    # TODO: Delete temporary renaming
+    dataset = dataset.rename(dict(tau_e_scaling="energy_confinement_scaling")).drop_vars(("fusion_reaction", "inductive_plasma_current"))
+    reference_dataset = reference_dataset.drop_vars(("fusion_reaction",))
+
```

Comment:
Is this only temporary to show that the tests otherwise pass?

Reply:
Yes, I wanted to make sure that the regression result doesn't change at all

Reply:
Update regression in latest commit